lib/typescript/render/components/Carousel/index.tsx

```diff
@@ -48,11 +48,14 @@ export const Carousel = ({children}) => {
     const resizeObserver = new ResizeObserver(() => {
       resetScrollButtons();
     });
-    resizeObserver.observe(carouselChildren.current);
-    resetScrollButtons();
-    carouselChildren.current.addEventListener('scroll', () => {
+
+    if (carouselChildren && carouselChildren.current) {
+      resizeObserver.observe(carouselChildren.current);
       resetScrollButtons();
-    });
+      carouselChildren.current.addEventListener('scroll', () => {
+        resetScrollButtons();
+      });
+    }
```

Comment:
fixed this in chat plugin carousel because got error in the UI because of the ref